examples/required-labels/template.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  creationTimestamp: null
+  name: requiredlabels
+spec:
+  crd:
+    spec:
+      names:
+        kind: RequiredLabels
+      validation:
+        openAPIV3Schema:
+          properties:
+            labels:
+              items:
```

Comment:
Looking at the schema, I think this is supposed to be type: array and items: string on the same indentation level.

```yaml
        openAPIV3Schema:
          properties:
            labels:
              type: array
              items: string
```

Reply:
It's not. The `items` key is a `*apiextensionsv1beta1.JSONSchemaPropsOrArray` which has a lot more options than a string of the item type. See also https://github.com/open-policy-agent/gatekeeper/blob/master/example/templates/k8srequiredlabels_template.yaml and https://github.com/open-policy-agent/gatekeeper-library/blob/master/library/general/requiredprobes/template.yaml

Reply:
Huh, you're right. I pulled that from https://github.com/open-policy-agent/gatekeeper#constraint-templates so maybe they both work or their documentation is dated. Carry on!